microshift_storage/volume-snapshots-microshift.adoc

```diff
@@ -0,0 +1,97 @@
+:_content-type: ASSEMBLY
+[id="working-with-volume-snapshots-microshift"]
+= Working with volume snapshots
+include::_attributes/attributes-microshift.adoc[]
+:context: volume-snapshots-microshift
+
+toc::[]
+
+This document describes how to work with volume snapshots to help protect against data loss by using the supported {product-title} logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. Familiarity with xref:../microshift_storage/understanding-persistent-storage-microshift.adoc#persistent-volumes_understanding-persistent-storage-microshift[persistent volumes] is required.
+
+A snapshot represents the state of the storage volume in a cluster at a particular point in time. Volume snapshots can also be used to provision new volumes. Snapshots are created as read-only logical volumes (LVs) located on the same device as the original data.
+
+With CSI volume snapshots, a cluster administrator can:
+
+* Create a snapshot of an existing persistent volume claim (PVC).
+* Backup a volume snapshot to a secure location.
+* Restore a volume snapshot as a different PVC.
+* Delete an existing volume snapshot.
+
+[IMPORTANT]
+====
+Support is only available for the logical volume manager storage (LVMS) CSI driver.
```

Comment:
In a later module (https://github.com/openshift/openshift-docs/pull/62557/files#diff-73ebfc6efde9d032adfc52fa08c552eb898f1ed708cb26608a7c39944ecffa82), you have a similar sentence describing LVMS CSI driver compatibility. I think the other usage is clearer.